src/services/users/actions.js

```diff
@@ -69,3 +69,7 @@ export function updateUser(data) {
 export function setPageNumber(pageNumber) {
   return setPageN(DOMAIN, pageNumber)
 }
+
+export function goToEdit(id) {
+  return push(`/my-account/users/edit/${id}`)
+}
```

Comment:
`push` is to perform navigation actions programmatically so, yes, if we are going to dispatch the `goToLogin` action as a response to a user click (and we don't need to do anything else in response to that click) I absolutely agree with you that we should use Links.